packages/wix-ui-tpa/src/components/Counter/Counter.tsx

```diff
@@ -19,6 +19,7 @@ export interface CounterProps extends TPAComponentProps {
   incrementAriaLabel: string;
   decrementAriaLabel: string;
   inputAriaLabel: string;
+  inputAriaLive?: 'off' | 'assertive' | 'polite';
```

Comment:
Maybe we can create an enum to reuse:
```tsx
export enum AriaLiveOptions {
  off = 'off',
  assertive = 'assertive',
  polite = 'polite',
}
```

Like this:
https://github.com/wix-private/wix-style-react/blob/master/packages/wix-ui-tpa/src/components/Image/types.ts#L35-L38


And then:
```suggestion
  inputAriaLive?: AriaLiveOptions
```

Reply:
done